docs/deploy/howto-add-aks-to-ps1.md

```diff
@@ -7,27 +7,27 @@ a Helm chart on top of a platform deployed through either `deploy.cmd` or `deplo
 
 ## Table of contents <!-- omit in toc -->
 
-* [Introduction](#introduction)
-* [Prerequisites](#prerequisites)
-* [Deployment Steps](#deployment-steps)
-  * [Create an AKS cluster](#create-an-aks-cluster)
-  * [Create a Public IP address](#create-a-public-ip-address)
-  * [Get credentials for kubectl](#get-credentials-for-kubectl)
-  * [Deploy ingress-nginx Helm chart](#deploy-ingress-nginx-helm-chart)
-  * [Deploy cert-manager Helm chart](#deploy-cert-manager-helm-chart)
-  * [Create ClusterIssuer resource](#create-clusterissuer-resource)
-  * [Stop App Service resources](#stop-app-service-resources)
-  * [Deploy azure-industrial-iot Helm chart](#deploy-azure-industrial-iot-helm-chart)
-    * [Using configuration in Azure Key Vault](#using-configuration-in-azure-key-vault)
-    * [Passing Azure resource details through YAML file](#passing-azure-resource-details-through-yaml-file)
-    * [Installing `azure-industrial-iot` Helm chart](#installing-azure-industrial-iot-helm-chart)
-      * [Installing the chart from GitHub repository](#installing-the-chart-from-github-repository)
-      * [Installing the chart from Helm chart repository](#installing-the-chart-from-helm-chart-repository)
-  * [Check status of deployed resources](#check-status-of-deployed-resources)
-  * [Enable Prometheus metrics scraping](#enable-prometheus-metrics-scraping)
-  * [Update Redirect URIs of web App Registration](#update-redirect-uris-of-web-app-registration)
-  * [Remove App Service and App Service Plan resources](#remove-app-service-and-app-service-plan-resources)
-  * [Access Engineering Tool and Swagger UIs](#access-engineering-tool-and-swagger-uis)
+- [Introduction](#introduction)
```

Comment:
@hansgschossmann, which VS Code extension are you using for this. I saw several use `*` as well and just want us to have consistent setup.

Reply:
I used Markdown All in One but the list marker was set to '-'. I changed it now to '*'.